### ISSUE TITLE
ModuleNotFoundError: No module named 'bigbang' on install

### ISSUE THREAD

**Issue by sbenthall** [datactive/bigbang#292]
On a fresh installation of BigBang, I get the following error on standard data collection:

```
(bigbang) sb@nothingness:~/projects/bigbang-multi/bigbang$ python bin/collect_mail.py -f examples/urls.txt
Traceback (most recent call last):
  File "bin/collect_mail.py", line 3, in <module>
    import bigbang.mailman as mailman
ModuleNotFoundError: No module named 'bigbang'
```

**Comment by sbenthall** [comment]
This appears to be a Python verson issue, see #226 .

Running

`python bin/collect_mail.py -f examples/urls.txt`

does not get this error

**Comment by nllz** [comment]
Don't see what the difference between the two command is. But I had the same issue, which I think related to pip installing Python 3.6 while being in a Anaconda Python 2.7 environment.

**Comment by sbenthall** [comment]
Quite right.
I somehow mistyped the very thing I was trying to point to:

`python2 bin/collect_mail.py -f examples/urls.txt`

**Comment by sbenthall** [comment]
README change is a stopgap.
The real fix is switching to Python 3 #226
